### DISCUSSION TITLE
Discord Server

### DISCUSSION THREAD

**Discussion by kevinbentley** [DescentDevelopers/Descent3#25] [Announcements]
Looks like the Reddit Descent Discord server is the place to be. A link from @JeodC is below.

**Comment by JeodC** [comment]
The most active Discord server I know of is the official reddit Descent server: https://discord.gg/sR5MxuG2

Invite link expires in a week. We've been talking this release up all day!

Looks like D3 specific development discussion has moved to Descent Developer Network: https://discord.gg/GNy5CUQ

**Comment by omega552003** [comment]
Admin of r/Descent, Here's a permanent invite link: https://discord.gg/FFt7hZuhne

Thank you so much for the source code! I know many people are excited to see what comes from this!
